src/libraries/System.IO.Hashing/ref/System.IO.Hashing.csproj

```diff
@@ -8,6 +8,10 @@
     <Compile Include="System.IO.Hashing.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

Comment:
nit: It would have been better to condition on a compatibility check / version check:

```xml
<ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">
```

to not lose the API on net7.0 when `$(NetCoreAppCurrent)` becomes net8.0. This just started failing in the PR that upgrades to net8.0: https://github.com/dotnet/runtime/pull/78354/checks?check_run_id=9846188314.

I will change this in our PR.

Reply:
Thanks